CHANGELOG.rst

```diff
@@ -1,6 +1,77 @@
 Changelog
 =========
 
+0.9.0 (2019-03-13)
+------------------
+
+Major features:
+
+- Support for running MLflow Projects in Docker containers. This allows users to include non-Python dependencies in their project environments and provides stronger isolation when running projects. See the `Projects documentation <https://mlflow.org/docs/latest/projects.html>`_ for more information. (#555, @marcusrehm; #819, @mparkhe; #970, @dbczumar)
```

Comment:
This allows users to include non-Python dependencies in their project -> This allows you to include non-Python dependencies in your project environments